code/pnpm-lock.yaml

```diff
@@ -0,0 +1,441 @@
+lockfileVersion: '9.0'
```

Comment:
please delete this lock file :)
my suggested changes on the gitignore will ignore this in the future

Reply:
https://pnpm.io/git#lockfiles

Reply:
this repo does not currently use pnpm though. it uses yarn currently and has the yarn lock file committed. we should only be using one manager.
I am happy to switch to pnpm (since I also like it better), but we can do that after this PR gets merged

Reply:
this file still needs to get deleted